src/input.js

```diff
@@ -21,6 +22,7 @@ class Input extends Component {
   constructor(props, context) {
     super(props, context)
     this.autocomplete = this.context[AUTOCOMPLETE_CONTEXT]
+    this.autoFocus = typeof this.context[MENU_CONTEXT] !== 'undefined'
```

Comment:
This changes the behavior from before... Now it will almost _always_ be `false` because the menu is often constructed _after_ the `input`. Could we just do `autoFocus={Boolean(this.menu)}`?